2023-risks/M8: Insufficient Cryptography

```diff
@@ -0,0 +1,106 @@
+---
+
+layout: col-sidebar
+title: "M8: Insufficient Cryptography"
```

Comment:
I'm entirely missing the topic of insecure hash functions. Could you please add a few words on that in relevant sections?